packages/google_sign_in/google_sign_in/example/android/gradle.properties

```diff
@@ -1,4 +1,4 @@
 org.gradle.jvmargs=-Xmx1536M
 android.enableR8=true
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
this line can be deleted since enableJetifier is false by default

Reply:
removed